NanoProd/config/skim_mu.yaml

```diff
@@ -65,7 +65,6 @@ selection: "
                           && abs(Electron_dz) < 0.2 && abs(Electron_dxy) < 0.045
                           && (Electron_mvaIso_WP90 || (Electron_mvaNoIso_WP90 && Electron_pfRelIso03_all < 0.5));
   auto prompt_muons = Muon_pt > 20 && abs(Muon_eta) < 2.4
-                      && abs(Muon_dxy) < 0.045
```

Comment:
What is the reason to remove dz and dxy for the prompt muon? Anyhow, if you decide to remove them, please do the same for prompt electrons to keep selections coherent.

Reply:
Removing these two cuts allows for additional studies on their effect on signal efficiency, if requested by MuonPOG for SF validation. For consistency, they are removed also for prompt electrons in the following commit.